plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/FeatureDevConstants.kt

```diff
@@ -13,3 +13,6 @@ const val CODE_GENERATION_RETRY_LIMIT = 3
 
 // The default retry limit used when the session could not be found
 const val DEFAULT_RETRY_LIMIT = 0
+
+// Max allowed size for a repository in bytes
+const val maxProjectSizeBytes: Long = 200 * 1024 * 1024
```

Comment:
nit: Can we have this constant name in ALL_CONSTANT, like MAX_PROJECT_SIZE_IN_BYTES? That seems the pattern for all constants in this file.

Reply:
sure, my bad